app/views/tutors/index.html.erb

```diff
@@ -1,28 +1,35 @@
-<h2>Tutors</h2>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Tutors</span>
```

Comment:
Size is fine; see screenshot. I think you may have made a mistake when you pulled to test.